### PR TITLE
Add V&V case, Mach 5 Shock-Wave Boundary-Layer Interaction

### PR DESCRIPTION
**Pull request by pcarruscag**
https://github.com/su2code/su2code.github.io/pull/77

### CODE REVIEW

**Comment by TobiKattmann** [comment]
And the addition of a mesh and restart files is done in https://github.com/su2code/TestCases/pull/101

**Review by TobiKattmann — APPROVED** [review]
Although marked as WIP I guess this is safe to approve ;) and of course here as well thanks for this vandv effort :bouquet: 

I really like your cfg "sections" ... very well made (I usually like the little header with author, date, etc. but I also see why that is not really necessary)

**Comment by aeroamit** [comment]
@pcarruscag, Nice.....
I saw the results and cross checked with reference results, they seems to be matching. I also observed that you have changed the mesh (adapted+boundary layer continuation for top wall). 
Seems like mesh was the culprit earlier?? (though inlet still seems to have minor issue but doesn't matter)
Also see the attached experiment data file (in case you want to use)...
[Schulein_2004_expt.dat.gz](https://github.com/su2code/SU2/files/8799043/Schulein_2004_expt.dat.gz)


Regards
Amit

File: TestCases/vandv.py [hunk 1]
```diff
@@ -64,6 +64,22 @@ def main():
     bump_sst1994m.test_vals = [-13.069139, -10.325333, -5.558903, 0.024576, 0.004967] 
     test_list.append(bump_sst1994m)
 
+    # SWBLI SA
+    swbli_sa           = TestCase('swbli_sa')
+    swbli_sa.cfg_dir   = "vandv/rans/swbli"
+    swbli_sa.cfg_file  = "config_sa.cfg"
+    swbli_sa.test_iter = 20
+    swbli_sa.test_vals = [-11.033668, -10.516942, -11.54364, -10.141842, -14.305338, 0.002233, -2.66486, 2.786]
```
Comment (pcarruscag) [inline TestCases/vandv.py:72]:
```suggestion
    swbli_sa.test_vals = [-11.029255, -10.511982, -11.400926, -10.128471, -14.536798, 0.002233, -2.608466, 2.786]
```

Comment (pcarruscag) [inline TestCases/vandv.py:72]:
```suggestion
    swbli_sa.test_vals = [-11.029255, -10.511982, -11.400926, -10.128471, -14.536798, 0.002233, -2.608466, 2.786]
```

File: .github/workflows/regression.yml [hunk 2]
```diff
@@ -92,7 +92,7 @@ jobs:
         uses: docker://su2code/test-su2:20200303
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c swbli -s ${{matrix.testscript}}
```
Comment (pcarruscag) [inline .github/workflows/regression.yml:95]:
```suggestion
          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
```

File: TestCases/vandv/rans/swbli/config_sst.cfg [hunk 3]
```diff
@@ -0,0 +1,98 @@
+% ------------------------------- SOLVER -------------------------------- %
```
Comment (pcarruscag) [inline TestCases/vandv/rans/swbli/config_sst.cfg:1]:
```suggestion
% See https://su2code.github.io/vandv/swbli/
%
% ------------------------------- SOLVER -------------------------------- %
```

**Pull request merged**